base/templates/base/public_base.html

```diff
@@ -429,20 +429,46 @@ <h3>{{self.friendly_name}}Social Media</h3>
                                 {% endif %}
                             </ul>
                         </div>
-                    {% endif %}
-
-                    {% if libra %}
+			{% endif %}
+			
+			{% if libra %}
                         <div class="rightside-mod col-xs-12 col-sm-6 col-md-12">
                             <h3>Print</h3>
                             <div class="libra-link-wrap">
                                 <p><a href="{% pageurl libra %}">{{libra}}</a></p>
                             </div>
                         </div>
-                    {% endif %}
+			{% endif %}
 
-                    {% if contacts %}
+			{% if contacts %}
                         {% list_contacts_in_sidebar contacts 'Media Contact' %}
-                    {% endif %}
+			{% endif %}
+
```

Comment:
Regarding the if `self.category` clause below, this only shows on LibNewsIndexPage, correct? There are cases where these code bits need to be in the public_base template. This might be one of those times, I'm not certain, however, I just want to be sure that we tried adding it to the lib_news_index_page template first. It's quite possible you tried this and it didn't work out. No worries if that's the case.

Reply:
Right; this is only for LibNewsIndexPage.  I tried putting the html for the RSS link on the lib_news_index_page right-hand sidebar the first time around.  However, my understanding is that the public_base right-hand sidebar html gets inserted *after* the lib_news_index_page right-hand sidebar html.  David wanted the RSS feed link to appear at the bottom of the right-hand sidebar.  My sense was that those two factors were kinda forcing me to put the html for the RSS feed in the public_base template, as ugly as that is.  But maybe that's wrong.  What do you think?  It would be great to move this out of the public_base template if there's a nice way to do that.

Reply:
Yes, I'm sure that's right. I've had to do it with other modules as well. I just wanted to do due diligence and make sure I asked. It's fine as it is.